src/bindings/python/src/openvino/frontend/pytorch/decoder.py

```diff
@@ -185,7 +186,10 @@ def get_output_transpose_order(self, index: int) -> list:
         return []
 
     def get_subgraph_size(self) -> int:
-        return len(self.get_subgraphs()) if hasattr(self.graph_element, "blocks") else 1
+        try:
+            return len(self.get_subgraphs()) 
```

Comment:
I don't like using try-except here, could you just change the if statement to avoid this?